express-middlewares/src/server.js

```diff
@@ -0,0 +1,24 @@
+const express = require('express');
+
+const PORT = 3000;
+const app = express();
+
+const languages = ['Python', 'JavaScript', 'C', 'Java', 'Lisp'];
+const packages = ['express', 'nodemon', 'create-react-app', 'redux', 'react-router-dom'];
+
+app.use(express.json({ extended: false }));
+
+app.use((req, res, next) => {
+  const { path } = req;
+  const properPath = path.slice(1);
```

Comment:
I think you can use "originalUrl" here so that you won't need to add this much logic to get the "properPath".
Like so: const { originalUrl } = req
This will give you the properPath.

Reply:
Did not know that. Thanks for the suggestion!